### ISSUE TITLE
.NET Core (.NET Standard) NUnit Parallel Tests

### ISSUE THREAD

**Issue by mitchellmaler** [nunit/nunit#2606]
Hello, 

We have started to port code over from .NET Framework to .Net Core which includes our tests. So far I have not found a way to run the nunit tests in parallel like we did using the .NET Framework. Is this not support for the .NET Core Version?

**Comment by jnm2** [comment]
Please see https://github.com/nunit/nunit/issues/2542 for an explanation. I'll be working on adding a netstandard2.0 build tonight which will enable parallelism.

**Comment by jnm2** [comment]
Do you want parallelism with .NET Core 1.x, or can I close this as a duplicate of the .NET Core 2 issue?

**Comment by mitchellmaler** [comment]
That's awesome! Excited to hear the support is on it's way. 
Yea this was for .NET Core 2. So it is a duplicate. Not sure how I missed that one when I did my search.

**Comment by jnm2** [comment]
No sweat, GitHub search is the way it is. 😃
